tests/test_rule_in_container.sh

```diff
@@ -38,16 +52,18 @@ _arg_rule=
 _arg_name="ssg_test_suite"
 _arg_scenarios=".*"
 _arg_datastream=
+_arg_remediate_using=
 
 
 print_help()
 {
 	printf '%s\n' "Test a rule using the container backend."
-	printf 'Usage: %s [-n|--name <arg>] [-s|--scenarios <arg>] [-d|--datastream <arg>] [-h|--help] <rule>\n' "$0"
+	printf 'Usage: %s [-n|--name <arg>] [-s|--scenarios <arg>] [-d|--datastream <arg>] [-r|--remediate-using <REMEDIATION>] [-h|--help] <rule>\n' "$0"
 	printf '\t%s\n' "<rule>: The short rule ID. Wildcards are supported."
 	printf '\t%s\n' "-n, --name: Name of the test image (default: 'ssg_test_suite')"
 	printf '\t%s\n' "-s, --scenarios: Regex to reduce selection of tested scenarios (default: '.*')"
-	printf '\t%s\n' "-d, --datastream: Path to the datastream to test. Autodetected by default. (no default)"
+	printf '\t%s\n' "-d, --datastream: Path to the datastream to use in tests. Autodetected by default. (no default)"
+	printf '\t%s\n' "-r, --remediate-using: What to remediate with. Can be one of: 'oscap', 'bash' and 'ansible' (no default)"
```

Comment:
You write "no default" but when I do not supply the -r, it still does something... am I understanding the help text in a wrong way?

Reply:
That is confusing, I have specified `oscap`. The original message was trying to say that no means of remediation are selected by default, and that leads to the default remediation, in this case the `oscap` kind, to be used.